Makefile

```diff
@@ -49,6 +49,7 @@ lint: ## Lint the python and golang sources
 	golangci-lint run -v
 
 core-install:
+	npm install
```

Comment:
```suggestion
core-install: web-install
```

Reply:
The `web-install` is ran as part of the `make install` command already, but I get the intent here as I had the same confusion debugging this. I put the `npm install` in a spot that made it seem like the web was part of it, I now realize the best spot to put the `npm install` is during the `gen-client-ts` step